store/mockstore/mocktikv/mvcc.go

```diff
@@ -743,6 +744,27 @@ func (s *MvccStore) BatchResolveLock(startKey, endKey []byte, txnInfos map[uint6
 	return nil
 }
 
+// DeleteRange deletes all keys in the given range.
+func (s *MvccStore) DeleteRange(startKey, endKey []byte) error {
```

Comment:
MvccStore is obsolete, you should implement MVCCLevelDB.

Reply:
Already done. PTAL